kbase-extension/static/kbase/js/common/runClock.js

```diff
@@ -1,46 +1,38 @@
-define([
-    'bluebird',
-    'common/runtime',
-    'kb_common/html',
-    'common/format'
-], function (
+define(['bluebird', 'common/runtime', 'common/html', 'common/format'], (
```

Comment:
Moved this module to the `common` directory, plus ran ESLint and Prettier on it.